src/util.ts

```diff
@@ -9,6 +9,52 @@ export interface BabelProperty extends ESTree.MethodDefinition {
   decorators?: BabelDecorator[];
 }
 
+/**
+ * Returns if given node has a lit identifier
+ * @param {ESTree.Node} node
+ * @return {boolean}
+ */
+function hasLitIdentifier(
+  node:
+    | ESTree.Node
+    | ESTree.Expression
+    | ESTree.SpreadElement
+    | null
+    | undefined
+): boolean {
+  return node?.type === 'Identifier' && node?.name === 'LitElement';
+}
+
+/**
+ * Returns if the given node is a lit element by expression
+ * @param {ESTree.Node} node
+ * @return {boolean}
+ */
+function isLitByExpression(
+  node: ESTree.Node | ESTree.Expression | undefined | null
```

Comment:
same here as the comment further up (change to `ESTree.Node` and do the null checking outside)